File: Level1/10.문자열내마음대로정렬하기/10.문자열내마음대로정렬하기.py
def solution(strings, n):
    answer = []
    for i in range(len(strings)):
        for j in range(i+1,len(strings)):
            if strings[i][n]>strings[j][n]:
                temp = strings[i]
                strings[i] = strings[j]
                strings[j] = temp
            elif strings[i][n]==strings[j][n] and strings[i]>strings[j]:
                temp = strings[i]
                strings[i] = strings[j]
                strings[j] = temp
        answer.append(strings[i])
    return answer

File: Level1/10.문자열내마음대로정렬하기/10.문자열내마음대로정렬하기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<string> solution(vector<string> strings, int n) {
    vector<string> answer;
    for(int i = 0 ; i<strings.size() ; i++){
        for(int j = i+1 ; j<strings.size() ; j++){
            if(strings[i][n]>strings[j][n]){
                string temp = strings[i];
                strings[i] = strings[j];
                strings[j] = temp;
            }else if(strings[i][n]==strings[j][n] && strings[i]>strings[j]){
                string temp = strings[i];
                strings[i] = strings[j];
                strings[j] = temp;
            }
        }
        answer.push_back(strings[i]);
    }
    return answer;
}